config/default/common/config/metadata/stories/geostationary/step007.md

```diff
@@ -0,0 +1,3 @@
+The Clean Infrared layer is useful for detecting clouds all times of day and night and is quite useful in retrievals of cloud top height. It is used to identify and classify cloud and other atmospheric features, estimate cloud-top brightness temperature and cloud particle size, convective severe weather signatures, and hurricane intensity. This imagery is at 2 km resolution. Click on the "Play" button and watch Hurricane Nicolas on September 13 to 14 making landfall on Texas and Louisiana. Note the very low temperatures of the storm clouds (in magenta/white/black) when the hurricane is in the Gulf of Mexico and becomes warmer (red to green) as it makes landfall.
```

Comment:
Updated to "The Clean Infrared layer is useful for detecting clouds all times of day and night. It is quite useful in retrievals of cloud top height. It can also be used to identify and classify cloud and other atmospheric features, estimate cloud-top brightness temperature and cloud particle size, convective severe weather signatures, and hurricane intensity. "

Reply:
Sounds good but looks like there's still an "at" missing in the first sentence.

Reply:
I think "at" sounds weird?